<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurveSetting</class>
 <widget class="QWidget" name="CurveSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,8">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>曲线显示与设置</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>主题类型:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,2,1,1,2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>曲线类型:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxDataType"/>
          </item>
          <item>
           <widget class="QLineEdit" name="textBox1DCurveShowValue"/>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>°/GHz</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="2,2,1">
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QRadioButton" name="radioButton">
                 <property name="text">
                  <string>频率</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBox">
                 <property name="singleStep">
                  <double>0.020000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QRadioButton" name="radioButton_2">
                 <property name="text">
                  <string>方位</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
                 <property name="minimum">
                  <double>-180.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>180.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.100000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>xmin:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="xMinEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>xmax:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="xMaxEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>xstep:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="xStepEdit"/>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>自动</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>数据源类别:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxDataProcessedType"/>
          </item>
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>绘图:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxplottype"/>
          </item>
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>信号模式:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxSignalWorkStyle"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1,1,1,1,1,1">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>ymin:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="yMinEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>ymax:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="yMaxEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>ystep:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="yStepEdit"/>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>自动</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>y轴类型:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxYAxisDataType"/>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>模式:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_ScanMode"/>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxDataBatchRangeCaculateMode">
            <property name="text">
             <string>批量显示</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,1,1,1,1,1">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxDataShowWindow">
            <property name="text">
             <string>加窗</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxDatashowZeroPad">
            <property name="text">
             <string>补零</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textBoxshowFFT_Length"/>
          </item>
          <item>
           <widget class="QPushButton" name="ButtonDataAnalyzeShow">
            <property name="text">
             <string>更新</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
